### PR TITLE
feat(frontend): Add e2e test for primary index scan

### PR DESCRIPTION
**Pull request by GeorgKreuzmayr**
Update planner tests

Signed-off-by: Georg Kreuzmayr <g.kreuzmayr@gmail.com>

I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

Showing that there is a problem with sorting on index scan

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: e2e_test/batch/order/test_order_requirers_index.part [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+1 4 54
+10 -1 70
+10 -2 28
+10 0 35
+10 5 53
+5 2 50
+6 -5 29
+8 2 17
+9 -2 32
+9 4 13
```
Comment (GeorgKreuzmayr) [inline e2e_test/batch/order/test_order_requirers_index.part:34]: Apparently once we use bigger test cases, there is an issue with the order here

Comment (jon-chuang) [reply e2e_test/batch/order/test_order_requirers_index.part:34]: Should we create an issue for this?

File: src/frontend/planner_test/tests/testdata/index_selection.yaml [hunk 2]
```diff
@@ -495,32 +512,3 @@
     BatchExchange { order: [], dist: Single }
       BatchFilter { predicate: (t1.a > 1:Int32) AND (t1.b > 1:Int32) }
         BatchScan { table: t1, columns: [t1.a, t1.b, t1.c, t1.p], distribution: SomeShard }
-- sql: |
```
Comment (jon-chuang) [inline src/frontend/planner_test/tests/testdata/index_selection.yaml:498]: Why are we removing this big test case?

**Comment by lmatz** [comment]
Let's close this one as long time no activity. Feel free to reopen.

**Pull request closed without merging**